src/helpers/getCourseDuration.ts

```diff
@@ -0,0 +1,7 @@
+export const getCourseDuration = (minutes: number): string => {
+	const hourDuration = 60;
+	const hours = Math.floor(minutes / hourDuration);
+	minutes = minutes % hourDuration;
```

Comment:
Hmm, let's create a new variable here, changing function arguments can be dangerous, especially if it's an object